app/src/lib/editors/launch.ts

```diff
@@ -30,7 +30,9 @@ export async function launchExternalEditor(
   }
 
   if (editor.usesShell) {
-    spawn(`"${editorPath}"`, [`"${fullPath}"`], { ...opts, shell: true })
+    //wrap each shell argument in quotes as it may include paths with spaces 
+    const args = editor.shellArgs ? editor.shellArgs?.map(arg => `"${arg}"`) : []
```

Comment:
have assumed here that all arguments are paths, will need to rethink this as arguments maybe other things e.g integer - might have to wrap them when they are defined.